submissions/DaniaB24/html-css-popup/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Web Site</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+</head>
+
+<body>
+    <nav class="nav-menu">
+        <ul class="nav__list">
+            <li class="nav__item"><a href="#" class="nav__link">Mail</a></li>
+            <li class="nav__item"><a href="#" class="nav__link">Images</a></li>
+            <li class="nav__item">
+                <div class="b-menu">
+                    <input class="check__input" type="checkbox">
+                    <input class="check__input2" id="more-btn" type="checkbox">
+                    <label for="more-btn" class="extra__label">More</label>
+                    <ul class="menu-top">
+                        <li class="li_popup" tabindex="1">
+                            <p class="p_popup"><img src="./images/calendar.png" class="popup">Calendar</p>
```

Comment:
Could you please add an attribute `alt`  to all images?
The tag `p` should be used for the paragraph. If you want to wrap any element you can use the tag `div`, `span`. Could you please fix it throughout where it was added?

Reply:
Hi) 
please check again:)